src/components/work/details/details.utils.js

```diff
@@ -0,0 +1,316 @@
+                {person?.display}
+              </Text>
+            );
+          })}
+        </>
+      )
+  );
+}
+
+/**
+ * Main method for retrieving fields to show in details section on workpage.
+ * Configurable arrays for different materialtypes - the fieldsMap holds array of
+ * configured objects to be shown - the DEFAULT is the base and may be overwritten if desired.
+ *
+ * example: to overwrite the languages object in the DEFAULT array with another definition for
+ * works with materialType MUSIC make another array like this:
+ * MUSIC: [
+ *       {
+ *         languages: {
+ *           label: "label for MUSIC materialtypes",
+ *           value: "values for MUSIC materialtypes",
+ *         },
+ *       },
+ *     ],
+ * fields are shown in the order of the objects
+ *
+ * For more complicated valued you may pass a jsx parser and thus overwrite the
+ * default presentation like this:
+ *
+ * MUSIC: [
+ *       {
+ *         languages: {
+ *           label: "label for MUSIC materialtypes",
+ *           value: "values for MUSIC materialtypes",
+ *           jsxParser: MovieLanguageValues,
+ *         }
+ *       }
+ *       ]
+
+ * To disable a field from the DEFAULT array overwrite with an empty value eg:
+ *      {
+ *         languages: {
+ *           label: "hundeprut",
+ *           value: "",
+ *         },
+ *       },
+ *
+ * @param manifestation
+ * @param work
+ * @param context
+ * @returns {*[]}
+ */
+export function fieldsForRows(manifestation, work, context) {
+  const materialType = work?.workTypes?.[0] || null;
+
+  const fieldsMap = {
+    DEFAULT: [
+      {
+        languages: {
+          label: Translate({ ...context, label: "language" }),
+          value: parseLanguages(manifestation),
+        },
+      },
+      {
+        physicalDescriptions: {
+          label: Translate({ ...context, label: "physicalDescription" }),
+          value: parsePhysicalDescriptions(manifestation),
+        },
+      },
+      {
+        publicationYear: {
+          label: Translate({ ...context, label: "released" }),
+          value: manifestation?.edition?.publicationYear?.display || "",
+        },
+      },
+      {
+        contributors: {
+          label: Translate({ ...context, label: "contribution" }),
+          value: parseContributors(manifestation),
+        },
+      },
+      {
+        genre: {
+          label: Translate({ ...context, label: "genre/form" }),
+          value: parseGenreAndForm(work),
+        },
+      },
+      {
+        hundeprut: {
+          label: "hundeprut",
+          value: "",
+        },
+      },
+    ],
+    MUSIK: [
+      {
+        genre: {
+          label: Translate({ ...context, label: "genre/form" }),
+          value: "hest",
+        },
+      },
+    ],
+    MOVIE: [
+      {
+        contributors: {
+          label: "",
+          value: parseMovieContributors(manifestation),
+          jsxParser: MovieContributorValues,
```

Comment:
Det ville nok være rarere hvis alle havde en `jsxParser`, men at dem som ikke behøver det bare defaulter til værdien.

Så slipper du for at `Details.js` skal tjekke osv. om der er en parser :relaxed: Det ligner lidt hvordan `manifestationsParser.js` har gjort det!